BLSQ/iaso
# IA-3375: Export user role and project as name not as id in csv and xlsx user export

Explain what problem this PR is resolving
- The user csv export on userrole column doesn't fite the import template
Related JIRA tickets : [IA-3375](https://bluesquare.atlassian.net/browse/IA-3375)

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes
- Display user role as name not as id
- Display project name instead of id

## How to test
- Go in user management
- Export csv or xlsx file
- Check the user role and project columns

## Print screen / video
[Screencast from 2024-08-29 16-53-08.webm](https://github.com/user-attachments/assets/72d5debb-dfc3-49df-98e7-85454d26f2e5)


## Notes

Things that the reviewers should know: known bugs that are out of the scope of the PR, other trade-offs that were made.
If the PR depends on a PR in bluesquare-components, or merges into another PR (i.o. main), it should also be mentioned here


[IA-3375]: https://bluesquare.atlassian.net/browse/IA-3375?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
